fe/src/static/sass/base.scss

```diff
@@ -1,7 +1,53 @@
-$font-stack:    Helvetica, sans-serif;
-$primary-color: #333;
+/***********************************************
+            DEFAULT STYLES
+***********************************************/
+
+@import url('https://fonts.googleapis.com/css?family=Montserrat');
+$font-stack: 'Montserrat', Helvetica, sans-serif;
+
+$black: #191414;
+$white: #FFFFFF;
+$green: #1DB954;
+
+html, body {
+  width: 100%;
+  height: 100%;
+}
 
 body {
+  margin: 0;
+  background-color: $black;
   font: 100% $font-stack;
-  color: $primary-color;
+  font-weight: lighter;
+}
+
+.main-container {
+  width: 100%;
+  height: 100%;
+}
+
+.content {
+  width: 100%;
+  height: 90%;
+}
+
+.title {
+  width: 100%;
+  height: 10%;
+  text-align: center;
+  color: $white;
+  font-size: 3em;
+}
+
+.widget, .sliders {
+  width: 50%;
+  display: inline-block;
+  color: $white;
+  text-align: center;
+}
+
+@media screen and (orientation: portrait) {
+  .widget, .sliders {
+    width: 100%;
+  }
```

Comment:
This doesn't enforce the equal height constraint. Try this:
```css
@media screen and (orientation: portrait) {
  .widget, .sliders {
    width: 100%;
    height: 50%;
  }
}

@media screen and (orientation: landscape) {
  .widget, .sliders {
    width: 50%;
    height: 100%;
  }
}
```
And obviously remove line 43 since it's gonna get overwritten after the media queries execute.